openpype/plugins/publish/cleanup_explicit.py

```diff
@@ -22,6 +22,12 @@ class ExplicitCleanUp(pyblish.api.ContextPlugin):
     active = True
 
     def process(self, context):
+        # Instance should be integrated on a farm
+        if context.data.get("farm"):
```

Comment:
I have been adding more of these same chec in my fork of OP in the plugins that I know I don't want to run locally (i.e., extract_review, extract_color_transcode...) instead of doing the families removal that we do in certain places to accomplish the same... doing this I find it way more clear than the current "families transfer" logic that we have in `submit_publish_job` to add back the families to the publish job.

Example of a case were "review" family gets removed:
```python
                if "review" in instance.families:
                    # to skip ExtractReview locally
                    instance.families.remove("review")
```
Much better to just add the same on `ExtractReview`?